packages/evolution-backend/src/services/audits/auditChecks/checks/__tests__/segment/testHelper.ts

```
@@ -0,0 +1,40 @@
+import { Household } from 'evolution-common/lib/services/baseObjects/Household';
+import { Home } from 'evolution-common/lib/services/baseObjects/Home';
+import { Interview } from 'evolution-common/lib/services/baseObjects/interview/Interview';
+import { SegmentAuditCheckContext } from '../../../AuditCheckContexts';
+
+export const createMockSegment = (overrides: Partial<Segment> = {}, validUuid = uuidV4()) => {
+    return {
+        _uuid: validUuid,
+        mode: 'walk',
+        ...overrides
+    } as Segment;
+};
```

Biome (2.1.2)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)